src/routes/guest-route.js

```diff
@@ -0,0 +1,8 @@
+import express from "express";
+const router = express.Router()
+import {createGuest} from "../helpers/guest";
+import {guestValidator, guestValidation} from "../middlewares/validationHandler/guestValidator";
+
+router.post('/', guestValidator, guestValidation, createGuest);
```

Comment:
this route is not great. You can do better.

Reply:
I have modified it. Thanks.